srcpkgs/devour/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'devour'
+pkgname=devour
+version=11.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkgconf xdo"
```

Comment:
Why is `xdo` needed in `hostmakedepends`?

Reply:
Hi! 
It was a typo, an error while copy the template from other pkgs.
Fixed it